src/dr/app/beauti/sitemodelspanel/PartitionModelPanel.java

```diff
@@ -723,6 +726,10 @@ private void setupPanel() {
                 //Do nothing
                 break;
 
+            case DataType.TREE:
+                //DO all sorts of things I haven't figured out yet
```

Comment:
Hmm?

Reply:
lol! Sorry this was a note to myself from when I was first looking through the beauti code.

I eventually figured out that nothing should be done.  I'll update the comment.